toc_subprojects/project-reviews-subproject/governance-review-template-guide.md

```diff
@@ -0,0 +1,419 @@
+
+_Needs Work: project's governance is lacking and inadequate for its current
+level of maturity, and needs substantial work to overcome that_
+
+_In Crisis: project appears to have several outstanding, critical issues that
+require attention, clarity, and/or direct guidance from CNCF in order to return
+to a state of good health and standing._
+
+
+**Status:** Exemplary | Satisfactory | Mostly Satisfactory | Needs Work | In Crisis
+
+_Short paragraph summarizing the general state of project governance.
+In the event the project governance requires attention, notify the TOC liaison
+for their awareness._
+
+
+### Executing the Assessment
+
+_A brief description that details the timebox the assessment occurred and the
+individuals involved in the assessment._
+
+
+### Must-Fix Items
+
+**The following issues have been identified that need to be resolved before
+[project milestone or other requirement]:**
+
+_The items in the list should be summarized, have a prioritized ordering
+and are expected to be considered Critical or High,  these are explicit and
+implicit blockers to project advancement within the CNCF Matriculation process.
+For each item in this list, a corresponding detailed description should be
+supplied. Note that which items are required depends on the project's maturity
+level._
+
+_File an issue with the project for each item on this list, and link to it._
+
+
+### Points of Excellence
+
+**The following aspects of governance are exemplary, and can be referenced as
+examples for other projects to copy:**
+
+_List of governance aspects where the project is exceeding expectations, or any
+novelty in their approach to governance._
+
+
+### Areas for Improvement
+
+**Over the next year, the project should work on the following issues to improve
+its governance, these are considered non-blocking:**
+
+_This is a summarized listing of longer term improvement areas for the project.
+These items are strongly encouraged but not required for the project's
+maturity level. Fully detailed descriptions are found in the Finding Table.
+Items listed here should be in priority ordering and would be considered
+Medium and Low._
+
+
+## Review
+
+**The following review primarily consists of an audit on the project's
+self-assessment in their Graduation application.**
+
+_Link to Project Application with Governance and Community Sections_
+
+
+### Governance Summary
+
+_Short narrative describing the governance type of the project, some general
+information about its leadership, and the project's general status and
+maturity._
+
+_If the project has any unusual aspects to its governance, describe
+them here._
+
+_Link to the project's existing documents where applicable._
+
+
+### Governance Evolution
+
+**Governance has continuously been iterated upon by the project as a result of
+their experience applying it, with the governance history demonstrating
+evolution of maturity alongside the project's maturity evolution.**
+<br>
```

Comment:
```suggestion

```

Reply:
not sure why there is a <br> here when the markdown new line is fine.

Reply:
When you don't use a `<br>` and just a markdown newline it does 2 lines instead of 1. This way it looks like this:
<img width="1031" height="108" alt="image" src="https://github.com/user-attachments/assets/08e4b770-7151-4528-b6b4-1f25fb206052" />